catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/RtfQueryResponseAndMetacardTransformer.java

```diff
@@ -73,11 +98,24 @@ public BinaryContent transform(
       throw new CatalogTransformerException("Null result set cannot be transformed to RTF");
     }
 
+    List<String> attributeOrder =
```

Comment:
Currently, one comes straight from the UI, while the other comes through the UI backend service. I agree it's annoying they are different. Since it's an ill defined contract, my fear with refactoring is it would break something that I'm not aware of, since it existed this way already. I don't think there are good itests for these scenarios either.